src/dia.py

```diff
@@ -0,0 +1,49 @@
+#----------------------------------------------------------------------------
+# Este archivo dia.py contiene la implementaciÃ³n de la clase dia.
+#----------------------------------------------------------------------------
+# @author: Antonio Caño @eantoniocalo18 
+# @created_date: 29/11/2021
+# @version: '2.0'
+# ---------------------------------------------------------------------------
+
+
+
+class Dia:
+    """
+    Clase utilizada para representar un día trabajado
+    ...
+    Atributos
+    ----------
+   
+    Publicos
+    id: str
+        uuid o id que identifica cada día único

```

Comment:
¿En serio necesitáis un ID para representar un día? ¿No será suficiene con la fecha?